bin/pocs_shell

```diff
@@ -92,6 +92,21 @@ class PocsShell(Cmd):
         i = DriftShell()
         i.cmdloop()
 
+    def do_calibrate_mount(self, *arg):
```

Comment:
*arg seems unncessary.

Reply:
All of the `do_` commands have that. We don't use it here but I just included it for consistency. Can remove.